gateway/utils/graphql/graphql.go

```diff
@@ -221,6 +238,12 @@ func (graph *Module) execGraphQLDocument(ctx context.Context, node ast.Node, tok
 			return
 		}
 
+		if field.Name.Value == "_query" {
```

Comment:
I suggest we don't do this. Simply store the queries array in `root._query` and store `root` in `coreParentKey`